api/src/controllers/user.ts

```diff
@@ -178,7 +180,15 @@ const getUser = async (
   id: number,
   getAutoExec: boolean
 ): Promise<UserDetailsResponse> => {
-  const user = await User.findOne({ id })
+  const user = (await User.findOne(
+    { id },
+    `id displayName username isActive isAdmin ${
+      getAutoExec ? 'autoExec' : ''
+    } -_id`
```

Comment:
There's no need to exclude document's attributes.
As we are explicitly returning data.